interfaces/builtin/account_control_test.go

```diff
@@ -41,13 +42,25 @@ var _ = Suite(&AccountControlSuite{
 			Snap:      &snap.Info{SuggestedName: "core", Type: snap.TypeOS},
```

Comment:
I wonder if we should not go all the way of using `snaptest.MockInfo`, we had issues where we had impossible structure / data that would not validate and would only make test pass in fake conditions.

+1 for now but if you want to get this fixed I'll gladly review that as well.